website/docs/getting-started/setup/instance-management/README.mdx

```diff
@@ -0,0 +1,130 @@
+---
+
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Instance Management
+
+## Updating to the latest release
+
+You can perform a manual update of your Appsmith instance by running the following commands in the Appsmith installation directory.
+
+```bash
+// To restart appsmith without docker compose
+docker pull appsmith/appsmith-ce && docker restart appsmith
+
+// To restart appsmith with docker compose
+docker-compose pull && docker-compose up -d --force-recreate appsmith
+```
+
+## Checkpoint upgrade
+Upgrading a Docker container checkpoint to the latest version is a crucial step in maintaining the stability, security, and performance of the containerized applications. 
+
+You may need to manually map your instance to the required version, in this case, ```v1.9.2```. To do this, access your Appsmith instance and make the following change to the `docker-compose.yml` file:
+
+<Tabs groupId="operating-systems">
+  <TabItem value="CE" label="Community Edition" default>
+  
+    From:
+
+```bash
+services:
+  appsmith:
+    image: index.docker.io/appsmith/appsmith-ce
+    container_name: appsmith
+```
+
+To:
+
+```bash
+services:
+  appsmith:
+    image: index.docker.io/appsmith/appsmith-ce:v1.9.2
+    container_name: appsmith
+```
+
+
+</TabItem>
+   <TabItem value="BE" label="Business Edition">
+   
+From:
+
+```bash
+services:
+  appsmith:
+    image: index.docker.io/appsmith/appsmith-ee
+    container_name: appsmith
+```
+
+To:
+
+```bash
+services:
+  appsmith:
+    image: index.docker.io/appsmith/appsmith-ee:v1.9.2
+    container_name: appsmith
+```
+   
+   
+   </TabItem>
+</Tabs>
+
+Save this file, then run the following command on your shell to restart Appsmith.
+
+```
+docker-compose up -d
+```
+
+:::note
+In this case, version `1.9.2` was used. If required, you can also use a specific older version.
```

Comment:
I don't think this makes sense, currently, our only mandatory checkpoint is v1.9.2.
Users have to hit this checkpoint migration if they are going to >1.9.2, not when going back to <1.9.2